gooddata-flight-server/Makefile

```diff
@@ -0,0 +1,6 @@
+# (C) 2021 GoodData Corporation
```

Comment:
Nitpick: check copyright dates pls

Reply:
went through all copyrights and bumped them to current year